### PR TITLE
Remove PS Unrestricted requirement from Rspec2Resx.targets

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**
I don't understand why this is necessary.
It's also breaking my local env

### CODE REVIEW

File: sonaranalyzer-dotnet/src/Rspec2Resx.targets [hunk 1]
```diff
@@ -18,7 +18,6 @@
     <PropertyGroup>
       <PowerShellExe Condition="'$(PowerShellExe)'==''">%WINDIR%\System32\WindowsPowerShell\v1.0\powershell.exe</PowerShellExe>
     </PropertyGroup>
-    <Exec Command="$(PowerShellExe) -command &quot;if ((Get-ExecutionPolicy) -ne &apos;Unrestricted&apos;) {echo &apos;Powershell: error 0000: You have to run Set-ExecutionPolicy Unrestricted (In powershell32)&apos;} &quot; " />
     <Exec ConsoleToMsBuild="true"
       Command="$(PowerShellExe) -NonInteractive -ExecutionPolicy Unrestricted -command &quot;  &amp; { ..\..\..\scripts\rspec\rspec2resx.ps1 $(Rspec2Resx)  $(ProjectDir)../../rspec/$(Rspec2Resx) $(ProjectDir)RspecStrings.resx } &quot;"/>
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/src/Rspec2Resx.targets]: Even if you have removed the previous line, I'd expect this line to fail now as you don't have the right.

Comment (andrei-epure-sonarsource) [reply sonaranalyzer-dotnet/src/Rspec2Resx.targets]: locally I'm having failures if I don't delete the lines...

Comment (Evangelink) [reply sonaranalyzer-dotnet/src/Rspec2Resx.targets]: Interesting, what's your local policy level? Could you try a `git clean -dfx` and then rebuild to see if everything is working as expected?

Comment (andrei-epure-sonarsource) [reply sonaranalyzer-dotnet/src/Rspec2Resx.targets]:
```
PS C:\Workspace\sonar-dotnet> get-executionpolicy
Unrestricted
PS C:\Workspace\sonar-dotnet> .\scripts\build\dev-build.ps1 -restore -build -metadata

Build FAILED.

Powershell : error 0000: You have to run Set-ExecutionPolicy Unrestricted (In powershell32) [C:\Workspace\sonar-dotnet\sonaranalyzer-dotnet\src\SonarAnalyzer.VisualBasic\SonarAnalyzer.VisualBasic.csproj]
```

Comment (andrei-epure-sonarsource) [reply sonaranalyzer-dotnet/src/Rspec2Resx.targets]: and w/o the line it works

**Comment by Evangelink** [comment]
Fix #2223

**Pull request merged**